lib/host/include/keystone_user.h

```diff
@@ -32,61 +32,66 @@
 
 #define MDSIZE 64
 
+#if __riscv_xlen == 64
+typedef __u64 u_ptr_t; 
+#elif __riscv_xlen == 32
+typedef __u32 u_ptr_t;
+#endif
+
 struct runtime_params_t {
-  __u64 runtime_entry;
-  __u64 user_entry;
-  __u64 untrusted_ptr;
-  __u64 untrusted_size;
+  u_ptr_t runtime_entry;
```

Comment:
Oh, I think we could use uintptr_t which is exactly the same as u_ptr_t you just defined.
Could you change it to be uintptr_t?

Reply:
Revised.